src/porepy/grids/match_grids.py

```diff
@@ -509,7 +509,9 @@ def structured_refinement(
     5. Assemble the mapping.
 
     """
-
+    if g.dim == 0:
+        mapping = sps.csc_matrix((np.ones(1), (np.array([0]), np.array([0]))))
```

Comment:
```suggestion
        mapping = sps.csc_matrix((np.ones(1), ([0], [0])))
```
I think this is equivalent, and (maybe?) easier to read.